pages/wiki/[article]/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useMemo, useState } from 'react';
+import { GetStaticPaths, GetStaticProps, InferGetStaticPropsType } from 'next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { visit } from 'unist-util-visit';
+import markdownLinkExtractor from 'markdown-link-extractor';
+
+import { unified } from 'unified';
+import remarkParse from 'remark-parse';
+import remarkRehype from 'remark-rehype';
+import rehypeStringify from 'rehype-stringify';
+import rehypeExternalLinks from 'rehype-external-links';
+import remarkGfm from 'remark-gfm';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { getCountryVariant } from 'utils/locales';
+
+const WikiArticle = ({ markdown, title }: InferGetStaticPropsType<typeof getStaticProps>): JSX.Element => {
+  const [seoTitle, setSEOTitle] = useState(title);
```

Comment:
Do you need state for this? If the component does not rerender, no reason this cannot be just a normal variable.